### ISSUE TITLE
export 'safeLoad' (imported as 'YAMLDummy') was not found in './formats/yaml.js'

### ISSUE THREAD

**Issue by SerhiyP** [i18next/i18next-fs-backend#15]
## 🐛 Bug Report

When I try to build project with webpack config where **target: "node"** I have those errors
```js
ERROR in ./node_modules/i18next-fs-backend/esm/readFile.js 6:47-65
export 'safeLoad' (imported as 'YAMLDummy') was not found in './formats/yaml.js' (possible exports: )
 @ ./node_modules/i18next-fs-backend/esm/index.js 24:0-55 78:26-38 87:6-14 164:8-16
 @ ./server/i18n.js 8:0-41 25:9-16
 @ ./server/node.js 13:0-26 26:20-24

ERROR in ./node_modules/i18next-fs-backend/esm/writeFile.js 6:47-65
export 'safeLoad' (imported as 'YAMLDummy') was not found in './formats/yaml.js' (possible exports: )
 @ ./node_modules/i18next-fs-backend/esm/index.js 25:0-57 161:10-20
 @ ./server/i18n.js 8:0-41 25:9-16
 @ ./server/node.js 13:0-26 26:20-24

webpack 5.11.0 compiled with 2 errors in 3111 ms

```
## To Reproduce

[repo](https://github.com/SerhiyP/i18n-bug)

```js
yarn // to install dependencies
yarn run ssr:dev:build // to start build
```

## Expected behavior

Build should not fail with errors

## Your Environment

- *runtime version*: node v12, v14
- *i18next version*: 19.8.4
- *os*: Linux

**Comment by adrai** [comment]
May I ask why you need to webpack this?

**Comment by SerhiyP** [comment]
@adrai 
Thank you for fast replay
I need this to bundle a lot of dependencies in order to run it on server

It's part of react SSR where a lot of **import** and modern **JS** that not supported without **babel**

**Comment by adrai** [comment]
I think this is a limitation of webpack not beeing able to import via wildcard...
I have no idea how to solve this for you, sorry.

**Comment by SerhiyP** [comment]
ok
I will wait maybe somebody else can advise something

**Comment by SerhiyP** [comment]
Hello
Issue fixed in version 1.0.8

Thanks
